Dockerfile

```diff
@@ -60,4 +60,14 @@ RUN ls /app/public/ && \
 ARG COMMIT_SHA
 ENV COMMIT_SHA=${COMMIT_SHA}
 
+# Create non-root user and group with specific UIDs/GIDs
+RUN addgroup -S appgroup -g 20001 && adduser -S appuser -G appgroup -u 10001
+
+# Change ownership only for directories that need write access
+RUN mkdir /app/tmp
```

Comment:
we seem to specifically delete the tmp directory on L58.
If we want to keep it maybe change L58 to 
rm -rf node_modules tmp/*
and remove L67